### PR TITLE
Convert slideshows to pure CSS

### PR DESCRIPTION
**Pull request by sndrs**
Removes the all the slideshow JS and converts it to pure CSS. Means:
- no DOM manipulation for slideshows now
- no lazy loading needed, since slideshow images are hidden with CSS on mobile and so not loaded
- no more lazy-loading code needed on the site now
- bug introduced in #10272 is obviated

@piuccio I can't find any other instances of this being used, do you know any? is it ok?
### testing
- [ ] no images stop appearing

### CODE REVIEW

**Comment by OliverJAsh** [comment]
Look at all those JS deletions!

I'm going to test and ship this on @sndrs’ behalf, as he is away on holiday.

**Comment by ScottPainterGNM** [comment]
guns-replaced-thumbs-up-19

Good man, John D has a code front with a slideshow on it already to test on.

**Comment by OliverJAsh** [comment]
Now assigned to @johnduffell.

**Comment by johnduffell** [comment]
unfortunately the issue isn't fixed, although before it happened when you didn't hover, now it happens when you do hover

**Comment by piuccio** [comment]
I'm at the beach so I can't really test. I love the idea of everything in css, but I'm not wise enough to understand how the magic works. 

Things to keep in mind 

- number of images is variable between 1 and 5
- mobile breakpoint only download first image 
- small cards display the default image even if they have sideshow 
- __aceessibility__ it's possible to disable sideshows from /accessibility/preferences or something like that in case someone has seizures caused by flashing images 

Keep merged if you're fine with changes

**Comment by johnduffell** [comment]
* you should definitely enjoy the beach instead of testing ;)
* I think the problem with mobile breakpoint will happen, I think it'll just download all anyway.  We might have to do some cleverness again to stop that.  It'll download all of the images on page load regardless.
* I'll check the small cards thing and the accessibility thing.

**Comment by ScottPainterGNM** [comment]
i-go-to-the-beach-to-surf

;)

**Comment by piuccio** [comment]
:'D

If all images are download at page load time, it's much better for performance to deliver that extra javascript. The original code starts downloading images only when they are needed. 

What are we trying to solve? If there's a blank while transitioning images, a better solution is to hook on image load to start the transition.

**Comment by johnduffell** [comment]
@piuccio I think we and @sndrs should discuss this when everyone's around.  It hink there's a tradeoff between simplicity and control here, so we need to make sure we're happy with what we're losing if we go down the current route.  In the mean time I'll fix it up to work with the updated version.

File: static/src/stylesheets/module/facia/_item.scss [hunk 1]
```diff
@@ -308,6 +308,54 @@ $fc-item-gutter: $gs-gutter / 4;
     display: none;
 }
 
+.fc-item__slideshow {
+    img {
+        @include mq($until: tablet) {
+            &:nth-child(1n+2) {
+                display: none;
```
Comment (stephanfowler) [inline static/src/stylesheets/module/facia/_item.scss]: These `display:none` images will still be downloaded though, won't they?

Comment (johnduffell) [inline static/src/stylesheets/module/facia/_item.scss]: seems not - in chrome at least it only seems to download when it needs them - however that does cause some tearing as it fades in

**Pull request merged**